core/keyserver/validate.go

```diff
@@ -100,6 +103,34 @@ func validateListEntryHistoryRequest(in *pb.ListEntryHistoryRequest, currentRevi
 	return nil
 }
 
+// validateListUserRevisionsRequest checks the bounds on start and end revisions and returns an appropriate number of
+// revisions to return for this request.
+func validateListUserRevisionsRequest(in *pb.ListUserRevisionsRequest, offset, currentRevision int64) (int64, error) {
```

Comment:
can we not use int instead of int64 all throughout?  int will default to 64 bit on 64-bit machines.

Reply:
Using int64 eliminates most of the casting (which IMO makes it more readable), since the field type in the proto is int64.

Reply:
I don't think any proto scalar types map onto Golang's int: https://developers.google.com/protocol-buffers/docs/proto3#scalar